e2e/txserver/zeta_tx_server.go

```diff
@@ -382,6 +388,50 @@ func (zts ZetaTxServer) DeploySystemContractsAndZRC20(
 		return "", "", "", "", "", fmt.Errorf("failed to deploy btc zrc20: %s", err.Error())
 	}
 
+	// FIXME: config this
```

Comment:
I think it is fine to use constant instead of config.
These values are for local network so would not change

Reply:
https://github.com/zeta-chain/node/commit/eb814a91d89bee5be48174081af7338f76cb197a